docs/_Docs_GetStarted.md

```diff
@@ -45,32 +31,27 @@ If you intend to work with `glider-rc.1` on the Görli testnet, proceed with ins
 
 ### Installation
```

Comment:
This section is an almost exact duplicate of the repo [readme](https://github.com/JoinColony/colonyNetwork#install).

Reply:
I don't want to butcher this section before @gichiba and @ryanchristo agree?

Reply:
I think this section could be improved upon and trimmed down a bit but I also think it would be best if a developer was able to get started without needing to leave the page (moving from the docs to the readme and back to the docs).

Reply:
I am going to fight very very hard against any duplication of installation instructions and dependencies. If you want to keep and maintain it here, I will remove it from the `readme`.